docs/source/examples/Notebook/Importing Notebooks.ipynb

```diff
@@ -130,7 +130,7 @@
     "                                       \n",
     "        # load the notebook object\n",
     "        with io.open(path, 'r', encoding='utf-8') as f:\n",
-    "            nb = current.read(f, 'json')\n",
+    "            nb = read(f, current_nbformat)\n",
```

Comment:
This should be specified as `4` explicitly - the idea of requiring it as a parameter is that code like this won't break each time we update the format, because it will be translated back to the format the code understands. Using `current_nbformat` means it will break.